governance.md

```diff
@@ -0,0 +1,79 @@
+  - Decide when we need new admins & handle their onboarding
+    - Encourage new admins to join via the typeform: https://devrelcollective.typeform.com/to/W452V9
+    - Change their member status from `Member` to `Admin` at https://devrelcollective.slack.com/admin
+    - Walk them through the process of processing applications -- talk them through the steps and then “pair program” with them during their first time walking it through on their own.
+    - Set them up with email access (Confirmation codes now get sent to team@devrelcollective.fun).
+  - Host Admin meetings once/month (in addition to any ad hoc meetings) to cover any outstanding agenda items and touch base with the team.
+  - Point person for relaying decisions that the Admin Team have made to the rest of the Collective.
+  - Keep a loose eye on and answer any questions that come through on Twitter
+  - When taking time off for vacation, family emergencies, etc. (as you should!), designate an “Interim Monarch” to be the proxy in your absence.
+- **Removal:**
+  - Violation of any Code of Conduct, be it DRC or any other community or organisation (online or in-person), is grounds for immediate removal, no questions asked.
+  - Inability to fulfill the responsibilities, as outlined above, over a three month period is grounds for removal.
+
+## Admins
+- **What is it:** A defined tier of admins who are responsible for the day-to-day health and success of the DRC; collectively: the Admin Team.
+- **Length of time:** 6 month stint which can “renew” at the end of each period. 
+- **Prerequisites:**
+  - Has been actively involved in the DevRel Collective for at least 6 months.
+  - Actively takes the initiative to move projects forward and be actively involved in conversations.
+- **Responsibilities:**
+  - Each admin is the lead for a specific function within the DRC:
+    - Mentorship Program Creation & Maintenance
+    - Salary Survey 
+    - Tool Development & Maintenance (e.g. On-Call Bot)
+    - Escalation Point for Applications (e.g. Responding to emails & making exec decisions re: applications)
+       - Be the executive decision-maker / tie-breaker for any applications that need additional review beyond a second set of eyes.
+       - Triage & reply to emails that come into the team@devrelcollective.fun inbox
+  - Everyone rotates through “on call bot” on a weekly basis:
+    - Handling of all access requests
+    - Escalate as appropriate
+    - Handle PR requests as appropriate
+  - Be present and available for monthly meetings.
+  - When an admin decides to step down, they’re responsible to find a replacement for that function -- whether through referral or re-opening the admin form.
+- **Removal:**
+  - Violation of any Code of Conduct, be it DRC or any other community or organisation (online or in-person), is grounds for immediate removal, no questions asked.
```

Comment:
@mary-grace I love this in spirit -- as @quintessence said beautifully: "there's a power differential with admins & members" and Mary said "we need to hold ourselves to a higher standard." 100% with that. 

In practice, the unbounded context of this phrasing concerns me. I don't think all CoC are easily discovered, so I imagine we might only use this unevenly which I'm concerned about. Could we make this more knowable? 

Specifically, what about: 

```suggestion
  - Violation of our Code of Conduct is grounds for immediate removal, no questions asked. Violation with any code of conduct where Admins are participating, be it DRC or any other community or organisation (online or in-person), are also grounds for immediate removal.
```

In short, I think the effort should be in us maintaining a meaningful super set of a CoC for the project so the rules are well formed and discoverable.

LMK if that sounds reasonable or if other, preferable, wording comes to mind.